crates/weaver_forge/src/extensions/util.rs

```diff
@@ -131,6 +132,16 @@ pub fn acronym(acronyms: Vec<String>) -> impl Fn(&str) -> String {
     }
 }
 
+fn to_yaml(value: Value) -> Result<String, minijinja::Error> {
```

Comment:
You should take the `tojson` filter implemented in the standard `minijinja` library as an example. The method’s signature is slightly different, and more importantly, they address the compatibility of the output with HTML (pay attention to the last map function). We don’t need the pretty-printing part since YAML is always pretty-printed and we probably don't need the indent parameter too.

See the permalink below.

https://github.com/mitsuhiko/minijinja/blob/d1ce496e101239bdfba01e29f407f0a57b8e4aed/minijinja/src/filters.rs#L1018